### PR TITLE
Add FAQ-Stores.md, re-arrange FAQ-readme, Summary.md

### PR DESCRIPTION
**Pull request by bitcoinshirt**
This PR adds new file FAQ-Stores.md.
This is an important addition, since it can be used for [Tooltips in UX.](https://github.com/btcpayserver/btcpayserver/issues/382)
I did my best to cover all of the important options. That said, since these settings are important for merchants and can cause problems if adjusted inappropriately, I would like to ask for someone to review and make sure I did not explain something inaccurately.

Please feel free to make any necessary edits. I'm fine with that.

PR : 
1. Creates FAQ-Stores.md
2. Adds FAQ-Stores.md to Summary.md
3. Updates and re-arranges FAQ/readme

### CODE REVIEW

**Comment by NicolasDorier** [comment]
We need to open an issue to btcpayserver/btcpayserver we should add link to those parts from inside BTCPay.

**Comment by NicolasDorier** [comment]
Sorry... you did ahah

**Comment by bitcoinshirt** [comment]
@NicolasDorier Yeah, I'm the "issue guy".
I'll await for your review prior to merging it, since it's very important so don't want to miss anything.

File: FAQ/FAQ-Stores.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+## Invoice expires if the full amount has not been paid after ... minutes
+The invoice timer is set to 15 minutes by default. The timer is a protection mechanism against the volatility, since it locks the cryptocurrency amount according to the crypto to fiat rates. If the customer does not pay the invoice within the defined period, the invoice will be considered expired. The invoice is considered "paid"as soon as the transaction is visible on the blockchain (o-confirmations), but considered "complete" when it reaches the amount of confirmations the merchant defined (usually, 1-6). The timer is customizable.
+
+## Payment invalid if transactions fails to confirm ... minutes after invoice expiration
+If the customer pays the invoice, but it fails to get the defined number of confirmations within the set time period, it will be marked as "invalid". Merchant can then decicide wheather to accept the invoice afterwards manuallly or decline it and require additional payment from the customer. This is an additional protection mechanism against the volatility.
```
Comment (NicolasDorier) [inline FAQ/FAQ-Stores.md]: `decicide` typo

Comment (NicolasDorier) [reply FAQ/FAQ-Stores.md]: `whether` typo

Comment (britttttk) [reply FAQ/FAQ-Stores.md]: `manuallly` typo

File: FAQ/readme.md [hunk 2]
```diff
@@ -32,6 +52,33 @@ Common questions and issues that may occur during the initial sync of BTCPay.
 * [BTCPay Server keep showing that my node is always starting](FAQ-Synchronization.md#btcpay-server-keep-showing-that-my-node-is-always-starting)
 * [I already have a synced full node, can I use it with BTCPay?](FAQ-Synchronization.md#im-running-a-full-node-and-have-a-synched-blockchain-can-btcpay-use-it-so-that-it-doesnt-have-to-do-a-full-sync)
 
+## [Server Settings FAQ](FAQ-ServerSettings.md)
+
+### [Maintainance FAQ](FAQ-ServerSettings.md#maintainance-1)
```
Comment (NicolasDorier) [inline FAQ/readme.md]: `Maintainance` actually is spelled `maintenance`

File: FAQ/readme.md [hunk 3]
```diff
@@ -32,6 +52,33 @@ Common questions and issues that may occur during the initial sync of BTCPay.
 * [BTCPay Server keep showing that my node is always starting](FAQ-Synchronization.md#btcpay-server-keep-showing-that-my-node-is-always-starting)
 * [I already have a synced full node, can I use it with BTCPay?](FAQ-Synchronization.md#im-running-a-full-node-and-have-a-synched-blockchain-can-btcpay-use-it-so-that-it-doesnt-have-to-do-a-full-sync)
 
+## [Server Settings FAQ](FAQ-ServerSettings.md)
+
+### [Maintenance FAQ](FAQ-ServerSettings.md#maintainance-1)
```
Comment (NicolasDorier) [inline FAQ/readme.md]: name the anchor?

**Comment by bitcoinshirt** [comment]
I believe I fixed everything. If you're okay with context and there's no issues with anything I explained or any other typos, you can merge. 
After that I'll tackle #103 and create FAQ for Altcoins, after which I'll proof-read entire FAQ folder and update summary.md and that all should finally close #88

**Pull request merged**